cypress/support/commandsCreateSignal.js

```diff
@@ -11,13 +59,45 @@ export const selectAddress = address => {
     .trigger('click');
 };
 
-export const inputDescription = description => {
+export const setAddress = (searchAdress, selectAdress) => {
+  searchAddress(searchAdress);
+  cy.wait('@getAddress');
+  selectAddress(selectAdress);
+  cy.wait('@geoSearchLocation');
+};
+
+export const setDescription = description => {
   cy.get('textarea')
     .clear()
     .invoke('val', description)
     .trigger('input');
 };
 
+export const setEmailAddress = emailAddress => {
+  cy.url().should('include', '/incident/email');
+  cy.checkHeaderText('Wilt u op de hoogte blijven?');
+  cy.get(CREATE_SIGNAL.inputEmail).clear().type(emailAddress);
+  cy.clickButton('Volgende');
+};
+
+export const setPhonenumber = phoneNumber => {
+  cy.url().should('include', '/incident/telefoon');
+  cy.checkHeaderText('Mogen we u bellen voor vragen?');
+  cy.get(CREATE_SIGNAL.inputPhoneNumber).clear().type(phoneNumber);
+  cy.clickButton('Volgende');
+};
```

Comment:
The behaviour of these two function is different from the others, even though their names make it seem that they perform similar actions. Where the other function don't navigate to the next form page, these functions do. For consistency's sake, I would not let these functions perform more than one predictable action and take the navigation actions out.